hugo.yaml

```diff
@@ -51,6 +51,12 @@ params:
       hue: 30deg
       saturation: 62%
       lightness: 49%
+    # TODO: We should derive the theme shades from the ThemeColor so that we
+    # don't have to explicitly define every theme color shade.
+    #
+    # In the interim, you can create theme colors using the shoelace color
+    # token generator.
+    # https://codepen.io/claviska/full/QWveRgL
```

Comment:
tracking issue?

Reply:
https://github.com/ecoacoustics/microsite-template/issues/49

I have also added an inline comment linking to the issue